### ISSUE TITLE
Error: [libs/sql-schema-describer/src/sqlite.rs:502:21] index out of bounds: the len is 1 but the index is 1 

### ISSUE THREAD

**Issue by tomhoule** [prisma/prisma#12613]
We don't have a clear idea of what the cause could be.

Command: `prisma db pull`
Version: `3.11.1`
Binary Version: `1a2506facaf1a4727b7c26850735e88ec779dee9`
Report: https://prisma-errors.netlify.app/report/13715 
OS: `arm64 darwin 21.3.0`

JS Stacktrace:
```
Error: [libs/sql-schema-describer/src/sqlite.rs:502:21] index out of bounds: the len is 1 but the index is 1
    at ChildProcess.<anonymous> (/.../packages/datafest-browsing-history-chromium/node_modules/prisma/build/index.js:46144:30)
    at ChildProcess.emit (node:events:390:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
```

Rust Stacktrace:
```
   0: backtrace::backtrace::trace
   1: backtrace::capture::Backtrace::new
   2: user_facing_errors::Error::new_in_panic_hook
   3: user_facing_errors::panic_hook::set_panic_hook::{{closure}}
   4: std::panicking::rust_panic_with_hook
   5: std::panicking::begin_panic_handler::{{closure}}
   6: std::sys_common::backtrace::__rust_end_short_backtrace
   7: _rust_begin_unwind
   8: core::panicking::panic_fmt
   9: core::panicking::panic_bounds_check
  10: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  11: sql_schema_describer::sqlite::SqlSchemaDescriber::get_table::{{closure}}::{{closure}}
  12: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  13: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  14: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  15: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  16: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  17: <futures_util::future::either::Either<A,B> as core::future::future::Future>::poll
  18: <futures_util::future::future::Then<Fut1,Fut2,F> as core::future::future::Future>::poll
  19: tokio::runtime::task::harness::poll_future
  20: tokio::runtime::task::raw::poll
  21: std::thread::local::LocalKey<T>::with
  22: tokio::runtime::thread_pool::worker::Context::run_task
  23: tokio::runtime::thread_pool::worker::Context::run
  24: tokio::macros::scoped_tls::ScopedKey<T>::set
  25: tokio::runtime::thread_pool::worker::run
  26: <tokio::runtime::blocking::task::BlockingTask<T> as core::future::future::Future>::poll
  27: tokio::runtime::task::harness::Harness<T,S>::poll
  28: tokio::runtime::blocking::pool::Inner::run
  29: std::sys_common::backtrace::__rust_begin_short_backtrace
  30: core::ops::function::FnOnce::call_once{{vtable.shim}}
  31: std::sys::unix::thread::Thread::new::thread_start
  32: __pthread_deallocate

```

**Comment by Jolg42** [comment]
Same error message, different line https://github.com/prisma/prisma/issues/10856

**Comment by miguelff** [comment]
Closing as duplicate of https://github.com/prisma/prisma/issues/10856

Writing a comment there to check wether the occurrence subject of this issue is fixed by fixing #10856 or needs reopening.
